### ISSUE TITLE
Problema con i widget

### ISSUE THREAD

**Issue by emiliomalizia** [PorteAperteSulWeb/pasw2015#139]
Buongiorno, 
con la versione 4.1 di word press e 1.3 del tema pasw2015 non vengono attivati (e non vedo nulla nel front end) i widget nel personalizzatore (per un altro sito stesso hosting funziona tutto) premetto che ho avuto messaggi di errore dopo aver importato il file modello sito 2014 11  07.xml.

**Comment by cghellere** [comment]
Cavoli ancora alla 1.3, io ti consiglierei di aggiornare il tema alla 1.6.6 ci sono nuove funzioni e codice migliorato. Per l'errore durante l'import xml mi pare di ricordare che qualche errore ci sia se non è attivo il modulo destinatari, e ultimamente dovrebbe dare errori sull'importazione dei media in quanto il sito di riferimento manca. Prova ad aggiornare.

**Comment by renatadurighello** [comment]
Ciao Emilio, potresti dirci che tipo di errori ti dava il file xml?

**Comment by emiliomalizia** [comment]
Non trova le immagini da caricare, infatti non sono presenti all'indirizzo
indicato nel file xml.
Il 16/set/2015 13:43, "renatadurighello" <notifications@github.com> ha
scritto:

**Comment by renatadurighello** [comment]
vero: erano su un mio spazio e non ci sono più. Però è un tipo di problema che non dovrebbe dare conseguenze sul funzionamento. MA le pagine ci sono tutte? Perché non sempre ha funzionato correttamente con le installazioni più recenti. Credo sia ancora un punto di partenza valido, ma forse necessita di essere aggiornato.

**Comment by emiliomalizia** [comment]
Si, le pagine ci sono. Ma sarebbe opportuno inserire le img ed aggiornare
il tutto
Il 16/set/2015 13:55, "renatadurighello" <notifications@github.com> ha
scritto:

**Comment by renatadurighello** [comment]
Grazie per la segnalazione. Resta il problema dei widget. Hai la pagina di amministrazione bianca?

**Comment by emiliomalizia** [comment]
No, sono riuscito a risolvere inserendoli dalla sezione widget e non dal
personalizzatore.
Il 16/set/2015 14:01, "renatadurighello" <notifications@github.com> ha
scritto:

**Comment by socrat3** [comment]
Ho provato ad usare l'ultimo template 1.6.6 corretto. Non funziona correttamente sia durante installazione che durante l'uso dei menu e dei widget

mancano le immagini. Dove posso trovarle? vorrei poterli mettere su un mio server locale e cambiare il percorso nel file xml. Mi aiutate?

**Comment by renatadurighello** [comment]
Ho ritrovato la cartella upload/2009/04 con le immagini. Cosa facciamo?

**Comment by emiliomalizia** [comment]
Potrebbe caricarli su uno spazio gratuito come per esempio altervista.

Saluti

2016-01-12 18:19 GMT+01:00 renatadurighello <notifications@github.com>:

> Ho ritrovato la cartella upload/2009/04 con le immagini. Cosa facciamo?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PorteAperteSulWeb/pasw2015/issues/139#issuecomment-170980565>
> .
>



-- 
*Ing.* Emilio Malizia
*Tel.:* 0984.1801034
*Mobile:* 347.5017590 - 345.7058385
*Fax: *0984.1801137
*P. Iva: *02991650785
*Skype:* emilio.malizia
*E-mail: *emiliomalizia@gmail.com
*PEC:* studiomalizia@pec.it



*Ai sensi del D.lgs 196/03 sulla tutela dei dati personali, le informazioni
contenute nella presente comunicazione e i relativi allegati sono
strettamente riservate ed esclusivamente indirizzate al destinatario
indicato (o alla persona responsabile incaricata di rimetterlo al
destinatario). La diffusione, distribuzione e/o copiatura del documento
trasmesso da parte di qualsiasi soggetto diverso dal destinatario è
proibita, sia ai sensi dell'art. 616 c.p., che ai sensi del D.Lgs. n.
196/2003. *
*Se avete ricevuto questo messaggio per errore, Vi preghiamo di
distruggerlo e di informarci immediatamente inviando un messaggio
all'indirizzo e-mail: emiliomalizia@gmail.com <emiliomalizia@gmail.com>
Grazie.*

*Per favore ricordate l’ambiente prima di stampare questa e-mail!*
